wsgi.py

```diff
@@ -5,4 +5,4 @@
 from packet import app
 
 if __name__ == "__main__":
-    app.run(host=app.config["IP"], port=int(app.config["PORT"]))
+    app.run(host=app.config["IP"], port=int(app.config["PORT"]), threaded=False)
```

Comment:
I'm not certain that this change is needed. We have it on a lot of other projects, so this was part of my debugging of timeout issues. I think bumping out the gunicorn timeout limit may solve the issue that this was meant to solve. But if nobody else has an issue with this change, I figure it's still okay